applications/ultrasound_postprocessing/tests/test_display.py

```
@@ -17,21 +17,22 @@
-    compress_grayscale,
-    ensure_rgba,
-    tensor_to_display,
-)
+    from ultra_post.core.display import (
+        DisplayCompressionSettings,
+        compress_color,
+        compress_grayscale,
+        ensure_rgba,
+        tensor_to_display,
+    )
+except Exception as exc:  # pragma: no cover - optional GPU dependency
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)